cg/src/utils/blockchain.js

```diff
@@ -69,7 +86,7 @@ async function getIsErased(subjectId) {
 async function getSubjectDataState(subjectId, processor = controllerAddress) {
   const quorumContract = await getContract();
   let result = await quorumContract.methods.getSubjectDataState(subjectId, processor).call();
-  return +result; //'+' coerces the returned value to a string
+  return +result; //'+' coerces the returned value to a number?
```

Comment:
Coerces to a number right?

Reply:
or `Number(result)` | `parseInt(result, 10)`?